components/style/color/colors.less

```
@@ -158,5 +158,18 @@
 @gold-9: color(~`colorPalette('@{gold-6}', 9) `);
 @gold-10: color(~`colorPalette('@{gold-6}', 10) `);
 
+// neutral color palette
+@gray-base: #bfbfbf;
```

For consistency with https://github.com/ant-design/ant-design-colors/blob/abc18fd97ecd5a45f81eef70bd250319a1736fd6/src/index.ts#L22
```suggestion
@gray-base: #666666;
```